.github/ISSUE_TEMPLATE/feature-request.yaml

```diff
@@ -0,0 +1,53 @@
+name: Feature Request
+description: A feature request for WildFly
+title: ""
+labels: ["feature"]
+projects: ["wildfly/7"]
+body:
```

Comment:
we could add a [markdown](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#markdown) type to give more contextual information for the user submitting the proposal (without polluting the created issue with this blurb of text).

This would give the user more info about what we are expecting for them when they fill the form.

Reply:
Sure, we could do that.